modules/openapi-generator/src/main/resources/php-symfony/composer.mustache

```diff
@@ -25,7 +25,7 @@
         "ext-mbstring": "*",
         "symfony/validator": "*",
         "jms/serializer-bundle": "^2.0",
-        "symfony/framework-bundle": "^2.3|^3.0|^4.1"
+        "symfony/framework-bundle": "^3.3|^4.1"
```

Comment:
Again the same discussion with the same conclusion as we had before in #2059, so I guess :+1:.

Reply:
Yes, this is the case when changing the version is justified.